featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -184,6 +184,42 @@ def label_func(df):
             feature_matrix['label_func']).values.all()
 
 
+def test_cfm_compose_approximate(es):
+    if not all(isinstance(entity.df, pd.DataFrame) for entity in es.entities):
+        pytest.xfail('dask does not support approximate')
+
+    def label_func(df):
+        return df['value'].sum() > 10
+
+    lm = cp.LabelMaker(
+        target_entity='id',
+        time_index='datetime',
+        labeling_function=label_func,
+        window_size='1m'
+    )
+
+    df = es['log'].df
+    df = to_pandas(df)
+    labels = lm.search(
+        df,
+        num_examples_per_instance=-1
+    )
+    labels = labels.rename(columns={'cutoff_time': 'time'})
+
+    property_feature = ft.Feature(es['log']['value']) > 10
+
+    feature_matrix = calculate_feature_matrix([property_feature],
+                                              es,
+                                              cutoff_time=labels,
+                                              approximate='1s',
+                                              verbose=True)
+    assert(type(feature_matrix) == pd.core.frame.DataFrame)
```

Comment:
could do `isinstance(feature_matrix, pd.core.frame.DataFrame)` if you'd like

Reply:
the compose LabelTimes object is a subclass of pd.DataFrame, so  an isinstance check would still return True if the feature matrix was a LabelTimes object